getStarted.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
```

Comment:
Never seen anyone make a getStarted.html page for this assignment. Very interesting choice. I like it.